astropy/tests/helper.py

```diff
@@ -30,7 +29,7 @@
 
 __all__ = ['raises', 'enable_deprecations_as_exceptions', 'remote_data',
            'treat_deprecations_as_exceptions', 'catch_warnings',
-           'assert_follows_unicode_guidelines', 'quantity_allclose',
```

Comment:
Can we do this on a bugfix branch? While importing with `*` is bad practice, this removal is an API change for those who does that.